pyemma/msm/tests/test_amm.py

```diff
@@ -0,0 +1,766 @@
+# This file is part of PyEMMA.
```

Comment:
I've inherited the TestMSMDoubleWell from test_msm now and overridden special cases. This removes a lot of redundancy.